From akpm@osdl.org Sat Oct 15 22:03:34 2005
Message-Id: <200510160502.j9G52nX8003445@shell0.pdx.osdl.net>
Subject: PCI: fix edac drivers for radisys 82600 borkage
From: akpm@osdl.org
Date: Sat, 15 Oct 2005 22:02:19 -0700


From: Andrew Morton <akpm@osdl.org>

I told you that the pci_ids.h cleanup was a bad idea ;)

Cc: Alan Cox <alan@redhat.com>
Signed-off-by: Andrew Morton <akpm@osdl.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 include/linux/pci_ids.h |    1 +
 1 file changed, 1 insertion(+)

--- gregkh-2.6.orig/include/linux/pci_ids.h
+++ gregkh-2.6/include/linux/pci_ids.h
@@ -1610,6 +1610,7 @@
 #define PCI_DEVICE_ID_SIIG_2S1P_20x_850	0x2062
 #define PCI_SUBDEVICE_ID_SIIG_QUARTET_SERIAL	0x2050
 
+#define PCI_VENDOR_ID_RADISYS		0x1331
 
 #define PCI_VENDOR_ID_DOMEX		0x134a
 #define PCI_DEVICE_ID_DOMEX_DMX3191D	0x0001
